src/jekyll/_includes/shared/highlight.liquid

```diff
@@ -1,5 +1,9 @@
-  </div>
-</div>
+{% comment %}
+Embeds a image, description and link to a specific Udacity course on the page.
+TODO: Apply and clean up styles
+TODO: Verify all strings are localized
+{% endcomment %}
+
```

Comment:
Is this only used for udacity pages? If so we should probably rename it from highlight.